go/vaas/pkg/broadcast_channel/broadcast_channel.go

```diff
@@ -0,0 +1,82 @@
+package broadcast_channel
```

Comment:
nit: The golang naming convention says: "Good package names are short and clear. They are lower case, with no under_scores or mixedCaps. They are often simple nouns [...]"

So I would suggest this:
```go
package broadcast

type Channel interface {}

type channel struct {}
```